lib/tasks/backfill_in_person_pending_at.rake

```diff
@@ -75,4 +75,67 @@ namespace :profiles do
 
     warn "backfill_in_person_verification_pending_at left #{profiles.count} rows"
   end
+
+  ##
+  # Usage:
+  #
+  # Print pending updates
+  # bundle exec rake profiles:backfill_deactivated_ipp_verification_cancelled
+  #
+  # Commit updates
+  # bundle exec rake profiles:backfill_deactivated_ipp_verification_cancelled UPDATE_PROFILES=true
+  #
+  task backfill_deactivated_ipp_verification_cancelled: :environment do |_task, _args|
+    ActiveRecord::Base.connection.execute('SET statement_timeout = 60000')
+
+    update_profiles = ENV['UPDATE_PROFILES'] == 'true'
+
+    InPersonEnrollment.where(status: [:expired, :cancelled, :failed]).
+      includes(:profile).
+      where.not(profile: { in_person_verification_pending_at: nil }).
+      find_in_batches do |batch|
+        batch.each do |enrollment|
+          profile = enrollment.profile
+          timestamp = profile.in_person_verification_pending_at
+
+          puts "#{profile.id},#{profile.deactivation_reason},#{timestamp}"
+          if update_profiles
+            profile.update!(
+              deactivation_reason: :verification_cancelled,
```

Comment:
Gina to do- should we be updating deactivation_reason? I would expect it to be already present- maybe we update it if it is nil?

Reply:
Resolving- we are fixing the prior enrollments